src/common/style/icon_color_css.ts

```diff
@@ -1,5 +1,23 @@
 import { css } from "lit-element";
 
+export interface StateColor {
+  state: string;
+  color?: string;
+}
```

Comment:
If we add this, the syntax should be the same as `state_image` <https://www.home-assistant.io/lovelace/picture-entity/#state_image>

Reply:
I had plans to add number ranges potentially in the future if you guys liked this, which would mean being similar to how conditions are defined.

Reply:
Wouldn't that be the same as `severity`?